server/plugin.go

```diff
@@ -83,7 +83,7 @@ type externalConfig struct {
 	DisplaySubscriptionNameInNotifications bool
 }
 
-const defaultMaxAttachmentSize = utils.ByteSize(10 * 1024 * 1024) // 10Mb
+const defaultMaxAttachmentSize = utils.ByteSize(100 * 1024 * 1024) // 100Mb
```

Comment:
My understanding from our last conversation was that the max file attachment size for Jira would match the max upload setting configured for MM server, unless the user explicitly overrides the setting in Jira.

Reply:
@wiggin77 I have made the changes as suggested by you, please have a look. Also, increased the `defaultMaxAttachmentSize` from 10 to 100, which is the same as the default value indicated by Mattermost for `maxFileSize`.